ibm/mas_airgap/roles/ocp_release_mirror/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Get Registry facts from environment variables if they have not been passed to the role
+registry_public_host: "{{ lookup('env', 'REGISTRY_PUBLIC_HOST') }}"
+registry_public_port: "{{ lookup('env', 'REGISTRY_PUBLIC_PORT') }}"
+registry_public_url: "{{ registry_public_host }}:{{ registry_public_port }}"
+
+# Release config
+openshift_release_version: "{{ lookup('env', 'OPENSHIFT_RELEASE_VERSION') }}"
+dir: "{{ lookup('env', 'DIR') }}"
```

Comment:
As before, can we use `log_dir` so purpose is clearer.

Reply:
ok